client/allocator/Cargo.toml

```diff
@@ -26,5 +26,5 @@ std = [
 	"sp-std/std",
 	"sp-core/std",
 	"sp-wasm-interface/std",
-	"log",
+	"log/std",
```

Comment:
```suggestion
```

Reply:
ahh about this: now the allocator is even in client. Should this crate really have a separate "std" feature? I can just remove that.

Reply:
The std feature is not needed. We should remove it IMO